packages/contentful/migrations/0100-create-landing-page.cjs

```diff
@@ -0,0 +1,434 @@
+require('dotenv').config();
+
+if (!process.env.SLUG_VALIDATION_APP_ID) {
+  console.log('No app ID specified in SLUG_VALIDATION_APP_ID; aborting.');
+  process.exit(1);
+}
+
+module.exports = async function(migration) {
+  // Update illustration to also allow text to be used by landing page
+  const illustration = migration.editContentType('illustration');
+
+  illustration
+    .createField('text')
```

Comment:
Should we add help text to inform users that text is only used in certain contexts?
For example using these illustrations in a cta won't even attempt to retrieve the text that is set on here.

Reply:
alternatively to keep the display and editor more alligned we could instead of adding this text to the illustration have a new intermediate contentType 'labeledIllustration' which would contain text and a link to an illustration.

Reply:
I agree to add a help text. I'm hesitant to add another labelledIllustration content type as it will result in more entries and more nesting (the editor having to open a window for the text, than for the illustration, than for the asset). I think the help text should make it clear enough.

Reply:
Help text added.